graylog2-rest-models/src/main/java/org/graylog2/rest/models/system/deflector/responses/DeflectorSummary.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.rest.models.system.deflector.responses;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+@JsonAutoDetect
+public abstract class DeflectorSummary {
+    @JsonProperty("is_up")
+    public abstract boolean isUp();
+
+    @JsonProperty("current_target")
+    public abstract String currentTarget();
```

Comment:
We should probably add [`@NotEmpty`](http://docs.jboss.org/hibernate/validator/5.1/api/org/hibernate/validator/constraints/NotEmpty.html) here.